managed-databases/postgresql-and-mysql/how-to/manage-manual-backups.mdx

```diff
@@ -17,40 +17,52 @@ categories:
 Manual backups are useful in cases when you need to have a backup of a specific state of the Database Instance. They can be used for Database Instances with local storage.
 
 <Macro id="iam-requirements" />
- 
+
 <Message type="requirement">
   - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
   - You have a [PostgreSQL or MySQL Database Instance](/managed-databases/postgresql-and-mysql/quickstart)
 </Message>
 
 ## How to create manual backups
```

Comment:
We can add more information here about Database Instance with block storage; 

It is possible to create manually Backups on Database Instance with block storage if the Database instance size is inferior or equal to 585G of storage

Reply:
added above